lib/bundler_api/update/job.rb

```diff
@@ -38,6 +39,8 @@ def insert_spec(spec)
     @db.transaction do
       rubygem_insert, rubygem_id = @db_helper.find_or_insert_rubygem(spec)
       version_insert, version_id = @db_helper.find_or_insert_version(spec, rubygem_id, @payload.platform, @payload.checksum, true)
+      info_checksum = @gem_info.info(spec.name)
```

Comment:
Nice catch. I wonder why this is passing on the tests? Most likely I'm missing a `Digest` call there.